lang/fr.js

```diff
@@ -1,25 +1,25 @@
 /* eslint quotes: 0 */
 
 export default {
-	"components.alert.close": "Fermer l'alerte",
-	"components.calendar.notSelected": "Pas sélectionné",
-	"components.calendar.selected": "Sélectionné",
+	"components.alert.close": "Fermer l’alerte",
+	"components.calendar.notSelected": "Non sélectionné(e)",
+	"components.calendar.selected": "Sélectionné(e).",
 	"components.calendar.show": "Afficher {month}",
 	"components.dialog.close": "Fermer cette boîte de dialogue",
 	"components.input-date.clear": "Effacer",
-	"components.input-date.openInstructions": "",
-	"components.input-date.setToToday": "",
+	"components.input-date.openInstructions": "Utiliser le format de date {format}. Utiliser la flèche vers le bas ou la touche d’entrée pour accéder au mini-calendrier.",
+	"components.input-date.setToToday": "Régler à aujourd’hui",
 	"components.input-date-time.date": "Date",
-	"components.input-date-time.time": "Heure",
+	"components.input-date-time.time": "Durée",
```

Comment:
This went from "Hour" -> "Duration"... neither of which seem right. @jeff-shepstone I think we want "Temps" here for "Time"?

Reply:
What's the context?

If it's time of day (the time is 12:35 PM), it would be the former value, Heure
If it's the amount of time that's passed or a duration, the new value Durée would be correct
Temps refers to the abstract, uncountable concept of time (e.g. time travel, time flies) and also means weather, so sometimes good to avoid it for that possible confusion

Reply:
Here's the context, it's a hidden accessible label for the second text box here which is for inputting a date and time:

![Screen Shot 2020-06-23 at 1 00 47 PM](https://user-images.githubusercontent.com/5491151/85432955-b2810600-b551-11ea-9c70-3d14127d97b6.png)

Reply:
In Google Calendar's similar component, they split it into two so it's not quite the same, but they use "Heure de début" and "Heure de fin":

![Screen Shot 2020-06-23 at 1 05 04 PM](https://user-images.githubusercontent.com/5491151/85433274-2d4a2100-b552-11ea-80e1-a2295dc5ed61.png)

I'll try to find some others...

Reply:
This is Google Tasks in Calendar, which only has a single time and they call it initially "Définir l'heure" but then I think incorrectly once you open it the label on the actual text input is "Heure de début" again.

![Uploading Screen Shot 2020-06-23 at 1.15.16 PM.png…]()

Regardless though it seems like the original "Heure" is more correct.